### ISSUE TITLE
Error linting file ended with space

### ISSUE THREAD

**Issue by doodlewind** [standard/standard#1036]
Hi, standard cli works great while it throws error linting following file:

[foo.js.zip](https://github.com/standard/standard/files/1588501/foo.js.zip)

```
➜  SideBar standard foo.js
standard: Unexpected linter output:

RangeError: Line number out of range (line 4 requested, but only 3 lines present).
    at SourceCode.getIndexFromLoc (/Users/ewind/.nvm/versions/node/v8.3.0/lib/node_modules/standard/node_modules/_eslint@3.19.0@eslint/lib/util/source-code.js:389:19)
    at Object.fix (/Users/ewind/.nvm/versions/node/v8.3.0/lib/node_modules/standard/node_modules/_eslint@3.19.0@eslint/lib/rules/no-multiple-empty-lines.js:116:52)
    at RuleContext.report (/Users/ewind/.nvm/versions/node/v8.3.0/lib/node_modules/standard/node_modules/_eslint@3.19.0@eslint/lib/rule-context.js:127:34)
    at Program:exit.allLines.reduce.concat.reduce (/Users/ewind/.nvm/versions/node/v8.3.0/lib/node_modules/standard/node_modules/_eslint@3.19.0@eslint/lib/rules/no-multiple-empty-lines.js:108:37)
    at Array.reduce (<anonymous>)
    at EventEmitter.Program:exit (/Users/ewind/.nvm/versions/node/v8.3.0/lib/node_modules/standard/node_modules/_eslint@3.19.0@eslint/lib/rules/no-multiple-empty-lines.js:93:22)
    at emitOne (events.js:120:20)
    at EventEmitter.emit (events.js:210:7)
    at NodeEventGenerator.applySelector (/Users/ewind/.nvm/versions/node/v8.3.0/lib/node_modules/standard/node_modules/_eslint@3.19.0@eslint/lib/util/node-event-generator.js:265:26)
    at NodeEventGenerator.applySelectors (/Users/ewind/.nvm/versions/node/v8.3.0/lib/node_modules/standard/node_modules/_eslint@3.19.0@eslint/lib/util/node-event-generator.js:294:22)

If you think this is a bug in `standard`, open an issue: https://github.com/standard/standard/issues
```

This file is ended with a space instead of a line break. Any ideas validating such input format?

**Comment by RyanFilho** [comment]
I downloaded the "standard" for the first time and I'm also having this problem:

```
/Development/book-by-isbn$ standard
standard: Unexpected linter output:

RangeError: Line number out of range (line 20 requested, but only 19 lines present).
    at SourceCode.getIndexFromLoc (/home/unknown/.nvm/versions/node/v8.9.3/lib/node_modules/standard/node_modules/eslint/lib/util/source-code.js:389:19)
    at Object.fix (/home/unknown/.nvm/versions/node/v8.9.3/lib/node_modules/standard/node_modules/eslint/lib/rules/no-multiple-empty-lines.js:116:52)
    at RuleContext.report (/home/unknown/.nvm/versions/node/v8.9.3/lib/node_modules/standard/node_modules/eslint/lib/rule-context.js:127:34)
    at Program:exit.allLines.reduce.concat.reduce (/home/unknown/.nvm/versions/node/v8.9.3/lib/node_modules/standard/node_modules/eslint/lib/rules/no-multiple-empty-lines.js:108:37)
    at Array.reduce (<anonymous>)
    at EventEmitter.Program:exit (/home/unknown/.nvm/versions/node/v8.9.3/lib/node_modules/standard/node_modules/eslint/lib/rules/no-multiple-empty-lines.js:93:22)
    at emitOne (events.js:121:20)
    at EventEmitter.emit (events.js:211:7)
    at NodeEventGenerator.applySelector (/home/unknown/.nvm/versions/node/v8.9.3/lib/node_modules/standard/node_modules/eslint/lib/util/node-event-generator.js:265:26)
    at NodeEventGenerator.applySelectors (/home/unknown/.nvm/versions/node/v8.9.3/lib/node_modules/standard/node_modules/eslint/lib/util/node-event-generator.js:294:22)

If you think this is a bug in `standard`, open an issue: https://github.com/standard/standard/issues

```

**Comment by feross** [comment]
Hey! Sorry for the delay in responding to this issue.

This crash was fixed in `standard` v11.0.0 which was released 3 months ago. ✨
